feat(changelog): console-added-console-search 2024-04-30 (#3135)
scaleway/docs-content: changelog/april2024/2024-04-30-console-added-console-search.mdx

---
title: Console Search
status: added
author:
    fullname: 'Join the #console-en channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-30
category: console
product: console
---

We are happy to announce that you can now find resources thanks to the new search in the console. 
Find them by their name or ID. More product integrations will come. 

> Check this article to see which products are available now.

Reviewer: ```suggestion
Refer to our [dedicated documentation](/console/account/how-to/use-the-search-bar/) to see which products are available now.
```
